### PR TITLE
Add SECURITY.md

### LINKED ISSUE #2096: Create a security policy
**Description of issue or feature request**:

Similar to [go-tuf](https://github.com/theupdateframework/go-tuf/issues/371), we should create a [security policy](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) so that researchers can properly disclose security issues.

### PR DESCRIPTION
**Pull request by mnm678**
Fixes #2096 

The text here is copied from the README for easier discovery. I'm happy to make any changes to this policy while we're looking at it.

### CODE REVIEW

**Review by lukpueh — CHANGES_REQUESTED** [review]
I suggest to remove the same text in README.md (prone to get out of sync) but keep the section and link to this new document.

File: README.md [hunk 1]
```diff
@@ -80,22 +80,7 @@ repo](https://github.com/theupdateframework/specification/issues).
...
-Optionally, reports that are emailed can be encrypted with PGP.  You should use
-PGP key fingerprint **E9C0 59EC 0D32 64FA B35F  94AD 465B F9F6 F8EB 475A**.
-
-Please do not use the GitHub issue tracker to submit vulnerability reports.
-The issue tracker is intended for bug reports and to make feature requests.
-Major feature requests, such as design changes to the specification, should
-be proposed via a
-[TUF Augmentation Proposal](https://theupdateframework.github.io/specification/latest/#tuf-augmentation-proposal-tap-support)
-(TAP).
+See [SECURITY.md](https://github.com/theupdateframework/python-tuf/blob/develop/SECURITY.md)
```
Comment (joshuagl) [inline README.md]: Nit: I think it's nicer to use relative links to docs in the same repo?
```suggestion
See [SECURITY.md](SECURITY.md)
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Security Issues and Bugs
+
+Security issues can be reported by emailing jcappos@nyu.edu.
+
+At a minimum, the report must contain the following:
+
+    Description of the vulnerability.
+    Steps to reproduce the issue.
```
Comment (joshuagl) [inline SECURITY.md]:
```suggestion
* Description of the vulnerability.
* Steps to reproduce the issue.
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+Optionally, reports that are emailed can be encrypted with PGP. You should use PGP key fingerprint E9C0 59EC 0D32 64FA B35F 94AD 465B F9F6 F8EB 475A.
+
```
Comment (joshuagl) [inline SECURITY.md]:
```suggestion
Optionally, reports that are emailed can be encrypted with PGP. You should use PGP key fingerprint **E9C0 59EC 0D32 64FA B35F 94AD 465B F9F6 F8EB 475A**.

```

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+Please do not use the GitHub issue tracker to submit vulnerability reports. The issue tracker is intended for bug reports and to make feature requests. Major feature requests, such as design changes to the specification, should be proposed via a TUF Augmentation Proposal (TAP).
```
Comment (joshuagl) [inline SECURITY.md]:
```suggestion
Please do not use the GitHub issue tracker to submit vulnerability reports. The issue tracker is intended for bug reports and to make feature requests. Major feature requests, such as design changes to the specification, should be proposed via a [TUF Augmentation Proposal](https://theupdateframework.github.io/specification/latest/#tuf-augmentation-proposal-tap-support) (TAP).
```

**Review by joshuagl — COMMENTED** [review]
Added some suggestions to match the original formatting.

File: README.md [hunk 5]
```diff
@@ -80,22 +80,7 @@ repo](https://github.com/theupdateframework/specification/issues).
...
-Optionally, reports that are emailed can be encrypted with PGP.  You should use
-PGP key fingerprint **E9C0 59EC 0D32 64FA B35F  94AD 465B F9F6 F8EB 475A**.
-
-Please do not use the GitHub issue tracker to submit vulnerability reports.
-The issue tracker is intended for bug reports and to make feature requests.
-Major feature requests, such as design changes to the specification, should
-be proposed via a
-[TUF Augmentation Proposal](https://theupdateframework.github.io/specification/latest/#tuf-augmentation-proposal-tap-support)
-(TAP).
+See [SECURITY.md](SECURITY.md)
```
Comment (lukpueh) [inline README.md]: Should we move `SECURITY.md` to `docs/`? All our `*.md` files are already there and[ GitHub will still find it](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository#about-security-policies):
  > To give people instructions for reporting security vulnerabilities in your project, you can add a SECURITY.md file to your repository's root, **docs**, or .github folder.

We'd have to adopt the change here...
```suggestion
See [SECURITY.md](docs/SECURITY.md)
```


 Related issue: #1849

**Review by lukpueh — CHANGES_REQUESTED** [review]
Please fix DCO and consider changing document location. Otherwise LGTM.

**Review by lukpueh — APPROVED** [review]
Thanks, Marina!

**Pull request merged**